test/controllers/cluster_infos_controller_test.rb

```diff
@@ -70,24 +70,24 @@ def teardown
 
   test "Should be error when wrong simulation ids are given" do
     get :show, id: @experiment.id.to_s, simulations: "Ala,1,2,3", using_em: 'false'
-    assert_response 500, "Response should be error, but it is: #{response.body}"
+    assert_response 412, "Response should be error, but it is: #{response.body}"
```

Comment:
I analogicznie 500 to :internal_server_error.
Generalnie:
https://www.omniref.com/github/rack/rack/1.5.2/symbols/Rack::Utils::SYMBOL_TO_STATUS_CODE?d=249779995&n=4#line=545
Te same nazwy po zamianie na underscore i symbol można używać zamiast kodów.